sdk/servicebus/service-bus/test/sendAndSchedule.spec.ts

```diff
@@ -380,10 +380,16 @@ describe("Sender Tests", () => {
 });
 
 describe("ServiceBusMessage validations", function(): void {
-  const sbClient = new ServiceBusClient(
-    "Endpoint=sb://a;SharedAccessKeyName=b;SharedAccessKey=c;EntityPath=d"
```

Comment:
Wouldn't removing the `EntityPath=d"` from the connection string have been enough to fix the test?

Reply:
Yes, that is enough.
However, since this code was not part of any before section, this gets executed before mocha reports the describe block as being run, which makes it non-trivial to know where the failure is.
![image](https://user-images.githubusercontent.com/10452642/92538103-b0deea00-f1f2-11ea-86ee-effbb62c01d3.png)


Making it part of the before section would help pinpoint this describe block if a failure happens.